packages/editor-ui/src/plugins/i18n/locales/en.json

```
@@ -1095,5 +1094,97 @@
 			"missingWorkflow": "Missing workflow",
 			"arrayEmpty": "Must have an array of nodes"
 		}
-	}
+	},
+	"404_NOT_FOUND_MESSAGE": "Oops, couldn’t find that",
+	"BASIC_INFORMATION": "Basic Information",
+	"CHANGE_PASSWORD": "Change Password",
+	"CHECK_INBOX_AND_SPAM": "Please check your inbox (and perhaps your spam folder)",
+	"CONFIRM_DATA_HANDLING_AFTER_DELETION": "What should we do with their data?",
+	"CONFIRM_USER_DELETION": "Are you sure you want to delete this invited user?",
+	"CURRENT_PASSWORD": "Current password",
+	"DELETE": "Delete",
+	"DELETE_CONFIRMATION_MESSAGE": "Type “delete all data” to confirm",
+	"DELETE_CONFIRMATION_TEXT": "delete all data",
+	"DELETE_USER": "Delete {user}",
+	"DELETE_WORKFLOWS_CREDENTIALS": "Delete their workflows and credentials",
+	"EMAIL": "Email",
+	"EMAIL_INVITES_SENT": "An invite email was sent to {emails}",
+	"EMAIL_INVITES_SENT_ERROR": "Could not invite {emails}",
+	"EMAIL_SENT_TO": "To {email}",
+	"ERROR": "Error",
+	"FINISH_ACCOUNT_SET_UP": "Finish account setup",
+	"FIRST_NAME": "First name",
+	"FORGOT_MY_PASSWORD": "Forgot my password",
+	"GET_RECOVERY_LINK": "Email me a recovery link",
+	"GO_BACK": "Go back",
+	"GO_TO_EDITOR": "Go to editor",
+	"INVALID_EMAIL_ERROR": "{email} is not a valid email",
+	"INVITE": "Invite",
+	"INVITE_NEW_USERS": "Invite new users",
+	"INVITE_RESENT": "Invite resent",
+	"INVITE_USER": "Invite user",
+	"INVITE_X_USER": "Invite {count} user",
+	"LAST_NAME": "Last name",
+	"LOADING": "Loading",
+	"LOGGING_IN_ERROR": "Problem logging in",
```

```suggestion
	"LOGIN_ERROR": "Problem logging in",
```
And wherever this is used.